public/locales/de/translation.json

```
@@ -227,7 +227,9 @@
     "hiddenColumnsVisible": "Ausgeblendete Spalten werden dir nun angezeigt!",
     "passwordCopied": "Board geschützt und Passwort kopiert!",
     "boardMadePublic": "Das Board ist nun öffentlich!",
-    "allParticipantsDone": "Alle Teilnehmer sind fertig."
+    "allParticipantsDone": "Alle Teilnehmer sind fertig.",
+    "noteToHiddenColumn": "Du hast gerade ein Karte einer ausgeblendeten Spalte hinzugefügt",
+    "noteToHiddenColumnButton": "Spalte für alle anzeigen"
```

The other toast messages use punctuation at the end, please use them too :)

```suggestion
    "noteToHiddenColumn": "Du hast gerade ein Karte einer ausgeblendeten Spalte hinzugefügt.",
    "noteToHiddenColumnButton": "Spalte für alle anzeigen"
```